src/payment/strategies/chasepay/chasepay-payment-strategy.ts

```diff
@@ -0,0 +1,228 @@
+            onCancel = () => {},
+            onError = (error: any) => {},
+            container,
+            isEditing,
+        } = options.chasepay;
+
+        this._onPaymentSelect = onPaymentSelect;
+        this._onCancel = onCancel;
+        this._onError = onError;
+        this._container = container;
+        this._isEditing = isEditing;
+
+        return this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(methodId))
+            .then(state => {
+                const storeConfig = state.config.getStoreConfig();
+
+                this._wepayRiskClient.initialize();
+                this._paymentMethod = state.paymentMethods.getPaymentMethod(methodId);
+
+                if (!this._paymentMethod || !this._paymentMethod.initializationData) {
+                    throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+                }
+
+                if (!storeConfig) {
+                    throw new MissingDataError(MissingDataErrorType.MissingCheckoutConfig);
+                }
+
+                return this._chasePayScriptLoader.load(this._paymentMethod.config.testMode)
+                    .then(({ ChasePay }) => {
+                        const { START_CHECKOUT, COMPLETE_CHECKOUT, CANCEL_CHECKOUT } = ChasePay.EventType;
+
+                        ChasePay.on(CANCEL_CHECKOUT, () => this._onCancel());
+                        ChasePay.on(COMPLETE_CHECKOUT, (payload: ChasePaySuccessPayload) => this._completeCheckout(payload, methodId)
+                            .then(() => this._onPaymentSelect())
+                            .catch(error => this._onError(error)));
+                        ChasePay.on(START_CHECKOUT, () => this._paymentInstrumentSelected(methodId));
+
+                        if (ChasePay.isChasePayUp && document.getElementById(this._container || '') && !this._isEditing) {
+                            ChasePay.insertBrandings({
+                                containers: [this._container],
+                                color: 'white',
+                            });
+                        }
+                    });
+            })
+          .then(() => super.initialize(options));
+    }
+
+    execute(orderRequest: OrderRequestBody, options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+        const { payment, ...order } = orderRequest;
+
+        if (!payment) {
+            throw new InvalidArgumentError('Unable to proceed because the customer has not signed into their ChasePay wallet account or selected a payment option.');
+        }
+
+        if (!this._paymentMethod || !this._paymentMethod.initializationData) {
+            throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+        }
+
+        const { initializationData } = this._paymentMethod;
+
+        if (!initializationData.paymentCryptogram || this._isEditing) {
+            return this._store.dispatch(this._orderActionCreator.submitOrder(order, options))
```

Comment:
You should only submit the order after the shopper has selected a payment option from the lightbox.